airflow/utils/orm_event_handlers.py

```diff
@@ -52,6 +55,40 @@ def set_mysql_timezone(dbapi_connection, connection_record):
             cursor.execute("SET time_zone = '+00:00'")
             cursor.close()
 
+        is_mariadb = engine.dialect.server_version_info and "MariaDB" in engine.dialect.server_version_info
+        if idle_session_timeout > 0:
```

Comment:
Not sure if this a best idea but how about:
```
if idle_session_timeout <= 0:
    return
```
in this way we may remove one indentation